cognite_toolkit/cdf_tk/load.py

```diff
@@ -412,18 +424,23 @@ def load_file(self, filepath: Path, ToolGlobals: CDFToolConfig) -> Group:
         for capability in raw.get("capabilities", []):
             for _, values in capability.items():
                 if len(values.get("scope", {}).get("datasetScope", {}).get("ids", [])) > 0:
-                    values["scope"]["datasetScope"]["ids"] = [
-                        ToolGlobals.verify_dataset(ext_id)
-                        for ext_id in values.get("scope", {}).get("datasetScope", {}).get("ids", [])
-                    ]
+                    if self.mode != "all_only":
+                        values["scope"]["datasetScope"]["ids"] = [
+                            ToolGlobals.verify_dataset(ext_id)
+                            for ext_id in values.get("scope", {}).get("datasetScope", {}).get("ids", [])
+                        ]
+                    else:
+                        # If we are running a clean, this is a no-op, but it needs to be valid
+                        values["scope"]["all"] = {}
+                        values["scope"].pop("datasetScope")
```

Comment:
if you pop all scopes, you need to skip the `capability`

Reply:
I'm popping only the datasetScope, not "all" that I set on the line before. I'm swapping out datasetscope with all scope.